agent/06_agent_create_cluster.sh

```diff
@@ -99,6 +123,11 @@ function attach_agent_iso() {
         agent_iso="${OCP_DIR}/agent.iso"
     fi
 
+    if [[ "${AGENT_PLATFORM_TYPE}" == "external" ]]; then
```

Comment:
It seems to me that this block may be logically better placed in step 05 than 06

Reply:
Agree. Updated